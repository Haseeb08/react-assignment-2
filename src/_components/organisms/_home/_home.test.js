import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Home from "./_home";

describe("Home component testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Home />);
    expect(input.toJSON()).toMatchSnapshot();
  });

  });
