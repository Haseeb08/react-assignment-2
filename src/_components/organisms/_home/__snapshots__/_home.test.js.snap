// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component testing Matches the snapshot 1`] = `
<div
  className="makeStyles-main-1"
>
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    Welcome
  </h2>
  <div
    className="makeStyles-image-2"
  >
    <img
      alt="profile pic"
      height="300"
      src="https://w1.pngwave.com/png/759/641/143/glasses-cartoon-face-eyewear-head-male-finger-png-clip-art-thumbnail.png"
      style={
        Object {
          "borderRadius": "100%",
        }
      }
      width="250"
    />
  </div>
  <h5
    className="MuiTypography-root MuiTypography-h5"
  >
    Abdul Haseeb
  </h5>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Incoming Fullstack engineer at Zemoso Technologies.
  </p>
</div>
`;
